docs/messaging/rabbitmq.md

```diff
@@ -12,8 +12,15 @@ The Semantic Conventions for [RabbitMQ](https://www.rabbitmq.com/) extend and ov
 
 ## RabbitMQ attributes
 
+### Destination name
+
 In RabbitMQ, the destination is defined by an *exchange* and a *routing key*.
-`messaging.destination.name` MUST be set to the name of the exchange. This will be an empty string if the default exchange is used.
+`messaging.destination.name` MUST be set according to the following rules:
+
+- Both *exchange* and *routing key* present and non-empty: `{exchange}:{routing key}`
+- Only *exchange* present and non-empty: `{exchange}`
+- Only *routing key* present and non-empty: `{routing key}`
+- Otherwise: `amp.default`
```

Comment:
typo?

```suggestion
- Otherwise: `amq.default`
```

also, do we need to set it? maybe just keep it empty if nothing was provided?

Reply:
Yep, it's a typo. I'm not sure of leaving it empty, I saw that in the RabbitMQ management UI it also shows `amq.default` so maybe it's nice to also have that very explicitly as part of the destination/span name.